package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "snowplow-analytics-sdk",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
0.2.0? Feels like a big change for a patch (plus changes to the Event interface).

Reply:
Is this a symptom that it should be a 1.x.x repo already?